app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,29 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.last_three_posts.includes(:comments)
+    @posts = @user.posts.includes(:comments).order(created_at: :desc)
+  end
+
+  def new
+    @post = Post.new
   end
 
   def show
     @post = Post.includes(:comments).find(params[:id])
   end
+
+  def create
+    @post = Post.new(post_params)
```

Comment:
- [ ] It would be better if the post is created on behalf of the currently loggedIn user as we have in the project requirements
![Screenshot from 2023-05-26 00-23-24](https://github.com/fithamlak/blog-app/assets/46388113/62138bd8-3b72-4350-93a2-e6161c5d141e)
It will look similar to this `@post = @current_user.posts.new(post_params)`

Reply:
- I was looking on it and it takes the current user from post_params author_id